_fixtures/cgotest.go

```diff
@@ -1,12 +1,15 @@
 package main
 
 /*
+#include <stdio.h>
```

Comment:
Can't reproduce until we have some C includes. Even new test will pass without this.
I have tested with all combinations between [oldTest, newTest, oldcgo, newcgo, cgo-with-only-runtime, cgo-with-only-header, cgo-with-header-runtime]
Tested with 1 variable at a time. newTest+cgo-with-header-with-runtime is required for fail test.